Keyboards/KeyboardsBase/Colors/UIColor+ScribeColors.swift

```diff
@@ -1,6 +1,22 @@
-//
-//  UIColor+ScribeColors.swift
-//
+/**
+ * Adds Scribe colors to the UIColor pool
```

Comment:
Minor change request on this, @Jag-Marcel, could we put a period at the end of all of the descriptions and just have one line after it instead of two?

Reply:
Sure! I'll get on it